app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] @benardop great work on your app, however, am unable to go past the login/signup page, am getting this error even when trying to create the user manually, had to remove some devise code to make it work, kindly ensure it works
![image](https://user-images.githubusercontent.com/61408860/211099752-39472550-5503-4c35-82cd-2b01e57bfe15.png)